### ISSUE TITLE
show.rownames not working in tab_df?

### ISSUE THREAD

**Issue by vanagpau** [strengejacke/sjPlot#699]
```{r}
data(iris)
describe(iris)
row.names(describe(iris))
tab_df(describe(iris), alternate.rows = TRUE, show.rownames = TRUE, use.viewer = TRUE)
```
row names are clearly present as can be seen from row.names command, but resulting table does not contain row names?

vars | n | mean | sd | median | trimmed | mad | min | max | range | skew | kurtosis | se
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
1 | 150 | 5.84 | 0.83 | 5.80 | 5.81 | 1.04 | 4.30 | 7.90 | 3.60 | 0.31 | -0.61 | 0.07
2 | 150 | 3.06 | 0.44 | 3.00 | 3.04 | 0.44 | 2.00 | 4.40 | 2.40 | 0.31 | 0.14 | 0.04
3 | 150 | 3.76 | 1.77 | 4.35 | 3.76 | 1.85 | 1.00 | 6.90 | 5.90 | -0.27 | -1.42 | 0.14
4 | 150 | 1.20 | 0.76 | 1.30 | 1.18 | 1.04 | 0.10 | 2.50 | 2.40 | -0.10 | -1.36 | 0.06
5 | 150 | 2.00 | 0.82 | 2.00 | 2.00 | 1.48 | 1.00 | 3.00 | 2.00 | 0.00 | -1.52 | 0.07

Is there a clash with packages maybe? These are what I have loaded:
library(psych)
library(tidyverse)
library(plyr)
library(alakazam)
library(car)
library(qqplotr)
library(gridExtra)
library(sjPlot)

**Comment by strengejacke** [comment]
Works fine for me:

``` r
library(sjPlot)
#> Learn more about sjPlot with 'browseVignettes("sjPlot")'.
data(iris)

tab_df(psych::describe(iris), alternate.rows = TRUE, show.rownames = TRUE, use.viewer = TRUE)
```

<table style="border-collapse:collapse; border:none;">

<tr>

<th style="border-top: double; text-align:center; font-style:italic; font-weight:normal; padding:0.2cm; border-bottom:1px solid black; text-align:left; ">

Row

</th>

<th style="border-top: double; text-align:center; font-style:italic; font-weight:normal; padding:0.2cm; border-bottom:1px solid black; ">

vars

</th>

<th style="border-top: double; text-align:center; font-style:italic; font-weight:normal; padding:0.2cm; border-bottom:1px solid black; ">

n

</th>

<th style="border-top: double; text-align:center; font-style:italic; font-weight:normal; padding:0.2cm; border-bottom:1px solid black; ">

mean

</th>

<th style="border-top: double; text-align:center; font-style:italic; font-weight:normal; padding:0.2cm; border-bottom:1px solid black; ">

sd

</th>

<th style="border-top: double; text-align:center; font-style:italic; font-weight:normal; padding:0.2cm; border-bottom:1px solid black; ">

median

</th>

<th style="border-top: double; text-align:center; font-style:italic; font-weight:normal; padding:0.2cm; border-bottom:1px solid black; col7">

trimmed

</th>

<th style="border-top: double; text-align:center; font-style:italic; font-weight:normal; padding:0.2cm; border-bottom:1px solid black; col8">

mad

</th>

<th style="border-top: double; text-align:center; font-style:italic; font-weight:normal; padding:0.2cm; border-bottom:1px solid black; col9">

min

</th>

<th style="border-top: double; text-align:center; font-style:italic; font-weight:normal; padding:0.2cm; border-bottom:1px solid black; 0">

max

</th>

<th style="border-top: double; text-align:center; font-style:italic; font-weight:normal; padding:0.2cm; border-bottom:1px solid black; 1">

range

</th>

<th style="border-top: double; text-align:center; font-style:italic; font-weight:normal; padding:0.2cm; border-bottom:1px solid black; 2">

skew

</th>

<th style="border-top: double; text-align:center; font-style:italic; font-weight:normal; padding:0.2cm; border-bottom:1px solid black; 3">

kurtosis

</th>

<th style="border-top: double; text-align:center; font-style:italic; font-weight:normal; padding:0.2cm; border-bottom:1px solid black; 4">

se

</th>

</tr>

<tr>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:left; ">

Sepal.Length

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; ">

1

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; ">

150

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; ">

5.84

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; ">

0.83

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; ">

5.80

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; col7">

5.81

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; col8">

1.04

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; col9">

4.30

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; 0">

7.90

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; 1">

3.60

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; 2">

0.31

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; 3">

\-0.61

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; 4">

0.07

</td>

</tr>

<tr>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:left; background-color:#f2f2f2; ">

Sepal.Width

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; background-color:#f2f2f2; ">

2

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; background-color:#f2f2f2; ">

150

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; background-color:#f2f2f2; ">

3.06

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; background-color:#f2f2f2; ">

0.44

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; background-color:#f2f2f2; ">

3.00

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; background-color:#f2f2f2; col7">

3.04

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; background-color:#f2f2f2; col8">

0.44

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; background-color:#f2f2f2; col9">

2.00

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; background-color:#f2f2f2; 0">

4.40

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; background-color:#f2f2f2; 1">

2.40

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; background-color:#f2f2f2; 2">

0.31

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; background-color:#f2f2f2; 3">

0.14

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; background-color:#f2f2f2; 4">

0.04

</td>

</tr>

<tr>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:left; ">

Petal.Length

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; ">

3

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; ">

150

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; ">

3.76

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; ">

1.77

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; ">

4.35

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; col7">

3.76

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; col8">

1.85

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; col9">

1.00

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; 0">

6.90

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; 1">

5.90

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; 2">

\-0.27

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; 3">

\-1.42

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; 4">

0.14

</td>

</tr>

<tr>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:left; background-color:#f2f2f2; ">

Petal.Width

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; background-color:#f2f2f2; ">

4

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; background-color:#f2f2f2; ">

150

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; background-color:#f2f2f2; ">

1.20

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; background-color:#f2f2f2; ">

0.76

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; background-color:#f2f2f2; ">

1.30

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; background-color:#f2f2f2; col7">

1.18

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; background-color:#f2f2f2; col8">

1.04

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; background-color:#f2f2f2; col9">

0.10

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; background-color:#f2f2f2; 0">

2.50

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; background-color:#f2f2f2; 1">

2.40

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; background-color:#f2f2f2; 2">

\-0.10

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; background-color:#f2f2f2; 3">

\-1.36

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; background-color:#f2f2f2; 4">

0.06

</td>

</tr>

<tr>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:left; border-bottom: double; ">

Species\*

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; border-bottom: double; ">

5

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; border-bottom: double; ">

150

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; border-bottom: double; ">

2.00

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; border-bottom: double; ">

0.82

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; border-bottom: double; ">

2.00

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; border-bottom: double; col7">

2.00

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; border-bottom: double; col8">

1.48

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; border-bottom: double; col9">

1.00

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; border-bottom: double; 0">

3.00

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; border-bottom: double; 1">

2.00

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; border-bottom: double; 2">

0.00

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; border-bottom: double; 3">

\-1.52

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; border-bottom: double; 4">

0.07

</td>

</tr>

</table>

``` r
tab_df(psych::describe(iris), alternate.rows = TRUE, show.rownames = FALSE, use.viewer = TRUE)
```

<table style="border-collapse:collapse; border:none;">

<tr>

<th style="border-top: double; text-align:center; font-style:italic; font-weight:normal; padding:0.2cm; border-bottom:1px solid black; text-align:left; ">

vars

</th>

<th style="border-top: double; text-align:center; font-style:italic; font-weight:normal; padding:0.2cm; border-bottom:1px solid black; ">

n

</th>

<th style="border-top: double; text-align:center; font-style:italic; font-weight:normal; padding:0.2cm; border-bottom:1px solid black; ">

mean

</th>

<th style="border-top: double; text-align:center; font-style:italic; font-weight:normal; padding:0.2cm; border-bottom:1px solid black; ">

sd

</th>

<th style="border-top: double; text-align:center; font-style:italic; font-weight:normal; padding:0.2cm; border-bottom:1px solid black; ">

median

</th>

<th style="border-top: double; text-align:center; font-style:italic; font-weight:normal; padding:0.2cm; border-bottom:1px solid black; ">

trimmed

</th>

<th style="border-top: double; text-align:center; font-style:italic; font-weight:normal; padding:0.2cm; border-bottom:1px solid black; col7">

mad

</th>

<th style="border-top: double; text-align:center; font-style:italic; font-weight:normal; padding:0.2cm; border-bottom:1px solid black; col8">

min

</th>

<th style="border-top: double; text-align:center; font-style:italic; font-weight:normal; padding:0.2cm; border-bottom:1px solid black; col9">

max

</th>

<th style="border-top: double; text-align:center; font-style:italic; font-weight:normal; padding:0.2cm; border-bottom:1px solid black; 0">

range

</th>

<th style="border-top: double; text-align:center; font-style:italic; font-weight:normal; padding:0.2cm; border-bottom:1px solid black; 1">

skew

</th>

<th style="border-top: double; text-align:center; font-style:italic; font-weight:normal; padding:0.2cm; border-bottom:1px solid black; 2">

kurtosis

</th>

<th style="border-top: double; text-align:center; font-style:italic; font-weight:normal; padding:0.2cm; border-bottom:1px solid black; 3">

se

</th>

</tr>

<tr>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:left; ">

1

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; ">

150

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; ">

5.84

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; ">

0.83

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; ">

5.80

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; ">

5.81

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; col7">

1.04

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; col8">

4.30

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; col9">

7.90

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; 0">

3.60

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; 1">

0.31

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; 2">

\-0.61

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; 3">

0.07

</td>

</tr>

<tr>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:left; background-color:#f2f2f2; ">

2

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; background-color:#f2f2f2; ">

150

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; background-color:#f2f2f2; ">

3.06

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; background-color:#f2f2f2; ">

0.44

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; background-color:#f2f2f2; ">

3.00

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; background-color:#f2f2f2; ">

3.04

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; background-color:#f2f2f2; col7">

0.44

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; background-color:#f2f2f2; col8">

2.00

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; background-color:#f2f2f2; col9">

4.40

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; background-color:#f2f2f2; 0">

2.40

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; background-color:#f2f2f2; 1">

0.31

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; background-color:#f2f2f2; 2">

0.14

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; background-color:#f2f2f2; 3">

0.04

</td>

</tr>

<tr>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:left; ">

3

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; ">

150

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; ">

3.76

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; ">

1.77

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; ">

4.35

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; ">

3.76

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; col7">

1.85

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; col8">

1.00

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; col9">

6.90

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; 0">

5.90

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; 1">

\-0.27

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; 2">

\-1.42

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; 3">

0.14

</td>

</tr>

<tr>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:left; background-color:#f2f2f2; ">

4

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; background-color:#f2f2f2; ">

150

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; background-color:#f2f2f2; ">

1.20

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; background-color:#f2f2f2; ">

0.76

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; background-color:#f2f2f2; ">

1.30

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; background-color:#f2f2f2; ">

1.18

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; background-color:#f2f2f2; col7">

1.04

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; background-color:#f2f2f2; col8">

0.10

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; background-color:#f2f2f2; col9">

2.50

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; background-color:#f2f2f2; 0">

2.40

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; background-color:#f2f2f2; 1">

\-0.10

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; background-color:#f2f2f2; 2">

\-1.36

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; background-color:#f2f2f2; 3">

0.06

</td>

</tr>

<tr>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:left; border-bottom: double; ">

5

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; border-bottom: double; ">

150

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; border-bottom: double; ">

2.00

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; border-bottom: double; ">

0.82

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; border-bottom: double; ">

2.00

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; border-bottom: double; ">

2.00

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; border-bottom: double; col7">

1.48

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; border-bottom: double; col8">

1.00

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; border-bottom: double; col9">

3.00

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; border-bottom: double; 0">

2.00

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; border-bottom: double; 1">

0.00

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; border-bottom: double; 2">

\-1.52

</td>

<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center; border-bottom: double; 3">

0.07

</td>

</tr>

</table>

<sup>Created on 2020-10-23 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

**Comment by vanagpau** [comment]
Yes I see. Weird. Any suggestions?

**Comment by strengejacke** [comment]
Can you post your session info? Are your packages up to date?

**Comment by vanagpau** [comment]
Thanks Dan

away from home atm, but it works fine on my (linux) laptop (home machine is windows).

will have a look again when I'm home.

Row | vars | n | mean | sd | median | trimmed | mad | min | max | range | skew | kurtosis | se
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
Sepal.Length | 1 | 150 | 5.84 | 0.83 | 5.80 | 5.81 | 1.04 | 4.30 | 7.90 | 3.60 | 0.31 | -0.61 | 0.07
Sepal.Width | 2 | 150 | 3.06 | 0.44 | 3.00 | 3.04 | 0.44 | 2.00 | 4.40 | 2.40 | 0.31 | 0.14 | 0.04
Petal.Length | 3 | 150 | 3.76 | 1.77 | 4.35 | 3.76 | 1.85 | 1.00 | 6.90 | 5.90 | -0.27 | -1.42 | 0.14
Petal.Width | 4 | 150 | 1.20 | 0.76 | 1.30 | 1.18 | 1.04 | 0.10 | 2.50 | 2.40 | -0.10 | -1.36 | 0.06
Species* | 5 | 150 | 2.00 | 0.82 | 2.00 | 2.00 | 1.48 | 1.00 | 3.00 | 2.00 | 0.00 | -1.52 | 0.07

**Comment by vanagpau** [comment]
R version 4.0.2 (2020-06-22)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 18363)

Matrix products: default

locale:
[1] LC_COLLATE=English_United Kingdom.1252  LC_CTYPE=English_United Kingdom.1252   
[3] LC_MONETARY=English_United Kingdom.1252 LC_NUMERIC=C                           
[5] LC_TIME=English_United Kingdom.1252    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] sjPlot_2.8.4    gridExtra_2.3   qqplotr_0.0.4   car_3.0-8       carData_3.0-4  
 [6] alakazam_1.0.2  plyr_1.8.6      forcats_0.5.0   stringr_1.4.0   dplyr_1.0.0    
[11] purrr_0.3.4     readr_1.3.1     tidyr_1.1.0     tibble_3.0.3    ggplot2_3.3.2  
[16] tidyverse_1.3.0 psych_1.9.12.31

loaded via a namespace (and not attached):
 [1] nlme_3.1-148      fs_1.4.2          lubridate_1.7.9   insight_0.8.5    
 [5] progress_1.2.2    httr_1.4.1        tools_4.0.2       backports_1.1.8  
 [9] sjlabelled_1.1.6  R6_2.4.1          DBI_1.1.0         lazyeval_0.2.2   
[13] colorspace_1.4-1  ade4_1.7-15       withr_2.2.0       tidyselect_1.1.0 
[17] prettyunits_1.1.1 mnormt_2.0.1      emmeans_1.4.8     curl_4.3         
[21] compiler_4.0.2    performance_0.4.7 cli_2.0.2         rvest_0.3.5      
[25] xml2_1.3.2        bayestestR_0.7.0  scales_1.1.1      mvtnorm_1.1-1    
[29] DEoptimR_1.0-8    robustbase_0.93-6 digest_0.6.25     minqa_1.2.4      
[33] foreign_0.8-80    rmarkdown_2.3     rio_0.5.16        pkgconfig_2.0.3  
[37] htmltools_0.5.0   lme4_1.1-23       dbplyr_1.4.4      rlang_0.4.7      
[41] readxl_1.3.1      rstudioapi_0.11   generics_0.0.2    jsonlite_1.7.0   
[45] zip_2.0.4         magrittr_1.5      parameters_0.8.0  Matrix_1.2-18    
[49] Rcpp_1.0.5        munsell_0.5.0     fansi_0.4.1       ape_5.4-1        
[53] abind_1.4-5       lifecycle_0.2.0   stringi_1.4.6     yaml_2.2.1       
[57] MASS_7.3-51.6     grid_4.0.2        blob_1.2.1        parallel_4.0.2   
[61] sjmisc_2.8.5      crayon_1.3.4      lattice_0.20-41   splines_4.0.2    
[65] ggeffects_0.15.0  haven_2.3.1       sjstats_0.18.0    hms_0.5.3        
[69] tmvnsim_1.0-2     knitr_1.29        pillar_1.4.6      igraph_1.2.5     
[73] boot_1.3-25       estimability_1.3  seqinr_4.2-4      effectsize_0.3.1 
[77] reprex_0.3.0      glue_1.4.1        packrat_0.5.0     evaluate_0.14    
[81] data.table_1.12.8 modelr_0.1.8      nloptr_1.2.2.2    vctrs_0.3.1      
[85] cellranger_1.1.0  gtable_0.3.0      assertthat_0.2.1  xfun_0.15        
[89] openxlsx_4.1.5    xtable_1.8-4      broom_0.7.0       coda_0.19-3      
[93] tinytex_0.24      statmod_1.4.34    ellipsis_0.3.1  

But I updated all packages and that's fixed it - thanks!

**Comment by strengejacke** [comment]
Ok, you have version 2.8.4. installed, the issue you see is fixed in version 2.8.5.
